### PR TITLE
Translate Searches for Numbers correctly

### PR DESCRIPTION
**Pull request by torbrenner**

### CODE REVIEW

File: core/cql/src/cql-translator.service.ts [hunk 1]
```diff
@@ -56,7 +56,9 @@ export class CqlTranslatorService implements QueryTranslator {
     // ["conditionLocalization", "exists from [Condition] C\nwhere C.bodySite.coding contains Code '{{C}}' from {{A1}}"],
     ["conditionLocalization", "exists from [Condition] C\nwhere C.bodySite.coding.code contains '{{C}}'"],
     ["conditionRangeDate", "exists from [Condition] C\nwhere year from C.onset between {{D1}} and {{D2}}"],
-    ["conditionRangeAge", "exists [Condition] C\nwhere AgeInYearsAt(FHIRHelpers.ToDateTime(C.onset)) between {{D1}} and {{D2}}"],
+    ["conditionRangeAge", "exists [Condition] C\nwhere AgeInYearsAt(FHIRHelpers.ToDateTime(C.onset)) >= {{D1}} and {{D2}}"],
```
Comment (patrick-skowronek) [inline core/cql/src/cql-translator.service.ts]: This does generate a CQL query that is not working. I guess it is missing a operator before the second placeholder.

**Review by patrick-skowronek — CHANGES_REQUESTED** [review]
The generated query does not work. Also I think the inputs in the HTML are in the wrong order.

File: core/cql/src/cql-translator.service.ts (hunk 1 above)
Comment (alexanderkiel) [reply core/cql/src/cql-translator.service.ts]: Yes `x >= y and z` doesn't work. What was the problem with `between`?

Comment (torbrenner) [reply core/cql/src/cql-translator.service.ts]: The templating didn't correctly replace either {C1} or {C2}. Strange that my local blaze instance executed the queries with `x >= 0`. Then I will try to make the templating work properly.

Comment (torbrenner) [reply core/cql/src/cql-translator.service.ts]: Sorry, took me longer to look again at that issue. The `x >= y  and z` stuff was actually just me doing a false commit. I had in my editor `x between y and z` but not staged for a commit.
It should now be corrected

**Review by patrick-skowronek — APPROVED** [review]
The query is now correct. However, the displayed data is very confusing with such a query. This should be discussed.

**Pull request merged**